style.css

```diff
@@ -0,0 +1,110 @@
+.wrapper {
+  width: 80%;
+  margin: auto;
+  border: dashed 1px red;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__logo {
+  padding: 40px 0;
+  font-family: 'ptserif-bold', arial, helvetica, Sans-serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 31px;
+  position: relative;
+  padding-left: 44px;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056ec;
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
+  opacity: .7;
+  border-radius: 50%
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+}
+
+.menu__item {
+  padding: 0 13px;
+  text-decoration: none;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  color: #000000;
+}
+ .menu__item:last-child {
+   padding-right: 0;
+ }
+
+ .menu__item:hover {
+   text-decoration: underline #0056EC;
+ }
+
+ .main-section {
+   display: flex;
+ }
+
+ .main-content {
+   margin-top: 105px;
+ }
+
+ .main-content__title {
+  font-family: PT Serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 66px;
+  line-height: 87px;
+  align-items: center;
+  color: #000000;
+ }
+
+ .main-content__subtitle {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 26px;
+  color: #000000;
+}
+
+.main-content__button {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  color: #FFF;
+  background: #0056ec;
+  border-radius: 4px;
+  padding: 10px 21px;
+}
+
+.image-content__image {
+  width: 770px;
+  height: 657px;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)